.github/workflows/comment-on-pull-request.yml

```diff
@@ -11,9 +12,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
```

Comment:
The checkout should be removed. It is not used and is a security risk.

Set this `pull_request_target` [best practices](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/)

Reply:
I've cleaned it up. I removed `synchronize` too